h/templates/client/help_panel.html

```diff
@@ -0,0 +1,43 @@
+<div class="help-panel">
+  <i class="close h-icon-close"
+    role="button"
+    title="Close"
+    ng-click="vm.onClose()"></i>
+
+  <header class="help-panel-title">
+    Help
+  </header>
+  <div class="help-panel-content">
+    <help-link
+      version="vm.version"
+      user-agent="vm.userAgent"
+      url="vm.url"
+      document-fingerprint="vm.documentFingerprint"
+      auth="vm.auth"
+      date-time="vm.dateTime">
+    </help-link>  
+    if you have any questions or want to give us feedback. 
+    You can also visit our <a class="help-panel-content__link" href="/docs/help" target="_blank"> help documents.</a>
```

Comment:
The `/docs/help` link will not work in the Chrome extension where the page is served from `chrome-extension://<ID>/viewer`. You need to prepend `{{vm.serviceUrl}}` where `vm.serviceUrl` is the URL of the Hypothesis service that the app is talking to.